### ISSUE TITLE
Stylistic Set: No backtick variants

### ISSUE THREAD

**Issue by pfitzseb** [cormullion/juliamono#27]
Would you consider adding a stylistic set (if that's the appropriate mechanism, that is) to disable the bigger single backtick:
![image](https://user-images.githubusercontent.com/6735977/90763151-7aded200-e2e6-11ea-8d72-af3d31f2c094.png)

**Comment by cormullion** [comment]
Which version are you on? I wonder if it was related to https://github.com/cormullion/juliamono/issues/8

**Comment by pfitzseb** [comment]
Huh, turns out I was on a really old version. Things actually are consistent on later versions (although I'm not quite sure I like how big the backticks are ;)).

**Comment by cormullion** [comment]
I can easily add a stylistic alternative for those - I do like large punctuation myself, but your eyes are probably sharper...

**Comment by pfitzseb** [comment]
Looking at it again I think the only complaint I have is that `` ` `` looks slightly "heavier" than `'` (seems to be slightly wider at the top?): 
![image](https://user-images.githubusercontent.com/6735977/90765151-bdee7480-e2e9-11ea-9482-6ff44705fa1e.png)

The size is something I'm I'll get used to in time :)

**Comment by cormullion** [comment]
What's your favourite font in this regard?

**Comment by pfitzseb** [comment]
I don't think I've seen a monospace font that handles this well (because technically it's an abuse of the grave accent?) yet -- the fact that JuliaMono handles `` ` `` the same as `'` or `"` regarding size and placements is really cool:
![image](https://user-images.githubusercontent.com/6735977/90766426-c8117280-e2eb-11ea-94ea-8e1fa3084306.png)
(Fira Mono)
![image](https://user-images.githubusercontent.com/6735977/90766441-d069ad80-e2eb-11ea-88ba-ac2b0eff31b3.png)
(JuliaMono)

So in summary: I'm totally happy with the current situation; adjusting the width of the non-diacritic `` ` `` would be the cherry on top :)

**Comment by simeonschaub** [comment]
I must admit, I wasn't the biggest fan of the large backticks at first as well, but I have now really come to enjoy them. I like how it makes them stand out a bit more, with other fonts, you're often not sure whether something is a backtick or a piece of dust on your display :grinning:. I think it still would be great to have the option to turn these off though.

**Comment by cormullion** [comment]
OK - I'll add it to my to-list anyway ... :) 

Fira Code is based on Fira, so it probably inherited the non-programming approach to some punctuation marks. Thank you for having such a detailed look!

**Comment by pfitzseb** [comment]
Yeah, I don't think I've ever seen a font show the non-diacritic differently.

Somewhat unrelated, but there also seems to be a hinting issue on Windows for certain font sizes that results in `` ` `` being unaligned with `'` and `"`:
![grafik](https://user-images.githubusercontent.com/6735977/90767758-fa23d400-e2ed-11ea-8967-2bae185ee12f.png)
![grafik](https://user-images.githubusercontent.com/6735977/90767890-322b1700-e2ee-11ea-9cb4-a8b8de74297a.png)

**Comment by cormullion** [comment]
Yes, probably Windows...

For the best and most accurate display, you need [one of these](https://www.apple.com/pro-display-xdr/). 🤣 Or any Apple retina display really... Your eyes will thank you, but your bank won't. Windows moves characters around in the process of hinting in order to get the most pixels lit up, so there are sometimes alignment problems, which vary according to font size anyway. I don't have many ways to fix it at the moment. :(

I've added another stylistic set, `SS07`.

<img width="1227" alt="Screenshot 2020-08-20 at 17 15 45" src="https://user-images.githubusercontent.com/52289/90797701-e5a10500-e308-11ea-9ae7-5cf8d141d31d.png">

It can go in the next release if you want it. It's called "Smaller grave" - I wanted "Shallow grave" but that was a bit grim...
